decidim-meetings/app/cells/decidim/meetings/meeting_card_metadata_cell.rb

```diff
@@ -23,6 +23,30 @@ def meeting_items
         [type, category_item, duration, comments_count_item, official, withdrawn_item]
       end
 
+      def meeting_items_for_map
+        [dates, type, official].compact_blank.map do |item|
+          {
+            text: item[:text],
+            icon: icon(item[:icon]).html_safe
+          }
+        end
+      end
+
+      def dates
+        text = if start_time.year != end_time.year
+                 "#{start_time.strftime("%d %b %Y")} - #{end_time.strftime("%d %b %Y")}"
+               elsif start_time.month != end_time.month || start_time.day != end_time.day
+                "#{start_time.strftime("%d %b")} - #{end_time.strftime("%d %b")}"
+               else
+                 "#{start_time.strftime("%H:%M")} - #{end_time.strftime("%H:%M")}"
+               end
+
+        {
+          text: ,
```

Comment:
```suggestion
          text: text,
```

Reply:
As far as I understand, linter will complaint about such syntax, won't it?

Reply:
Right now, it's pointing just the space after